packages/harmony/src/components/input/SelectablePill/SelectablePill.mdx

```diff
@@ -68,6 +68,12 @@ Only Large pills can have a leading icon.
 
 <Canvas of={SelectablePillStories.PillAsInput} />
 
+### Oversized pill
+
+This is a deprecated style used only in specific circumstances.
```

Comment:
would prefer not to have deprecated stuff in Harmony, but i suppose this is the only way if we want to pull out stems early. Since it's deprecated, I think we should not put this in Storybook

Reply:
great callout, ill hide it from storybook and add a comment in types that you should not use oversized